site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -448,7 +444,7 @@ user_registration_request = {
 Finally, let's specify our `client_response`.
 
 ```python
-client_response = client.register(None, user_registration_request)
+client_response = client.register(user_registration_request, None)
```

Comment:
incorrect order breaks re: comments on blog post

Reply:
This was an issue with anyone running pre 1.15.3, per https://github.com/FusionAuth/fusionauth-python-client/commit/ee9d6e83c03ab788ef1d891b1cf2938e82024558

So we should probably note that if you are running a version of the FusionAuth python client 1.15.3 or earlier, use `client.register(None, user_registration_request)` otherwise use the code you modified. Look at 2020-08-06-securely-implement-oauth-vuejs.md for callout examples.

Reply:
are these callouts  the blocks with the exclamation point/lightbulb icons?